src/modules/Laboratorio/Listar/actions/selectionStatus.js

```diff
@@ -1,12 +1,12 @@
 module.exports = function selectionStatus(status) {
```

Comment:
Essa variável é do tipo `boolean`. Nese caso, é melhor utilizar um nome que deixe claro que se trata de uma variável que só possa ser verdadeiro ou falso.

Quando você chama essa função no arquivo [Laboratorio.js](https://github.com/DCOMP-UFS/comunicalab-web/blob/2f97f1ff2c4071366a46ca6dc5dd7b94d7bea876/src/modules/Laboratorio/Listar/components/Laboratorio.js#L12), o nome da variável é `is_in_use`. Dá para utilizar esse mesmo nome.

```suggestion
module.exports = function selectionStatus(isInUse) {
```